src/core/public/doc_links/doc_links_service.ts

```diff
@@ -397,6 +397,8 @@ export class DocLinksService {
           functionbeat: `https://opensearch.org/docs/latest/downloads/beats/functionbeat`,
           winlogbeat: `${OPENSEARCH_WEBSITE_DOCS}`,
           siem: `${OPENSEARCH_WEBSITE_DOCS}`,
+          openSearchForum:
+            'https://forum.opensearch.org/t/feedback-experimental-feature-connect-to-external-data-sources/11144',
```

Comment:
try open but get `page not found`?

Reply:
I asked Kroosh about this link before. The link is correct. It's private until the feature goes live, in order to not drive attention to the forum post.